model/annotations/task_annotations.go

```diff
@@ -22,6 +22,8 @@ type TaskAnnotation struct {
 	Issues []IssueLink `bson:"issues,omitempty" json:"issues,omitempty"`
 	// links to tickets possibly related
 	SuspectedIssues []IssueLink `bson:"suspected_issues,omitempty" json:"suspected_issues,omitempty"`
+	// links to tickets created from the task using a custom web hook
+	CreatedIssues []IssueLink `bson:"created_issues,omitempty" json:"created_issues,omitempty"`
```

Comment:
Why do we need a separate structure for this (as opposed to just adding to issues)?

Reply:
A ticket filed from the task page is not necessarily categorized as an issue once it's created for the task. If they want to designate it as an issue or suspected issue, they'd then have to add it to one of those lists.